### PR TITLE
Added Competitive Programming Learning Path as a blog post

### PR DESCRIPTION
**Pull request by harshaldesai01**
#### Type of Change:

- [x] Blog

#### Description

#22

CZ now features Competitive Programming Learning Path.

### CODE REVIEW

File: _posts/CPLP.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
+
+
+***
+#   Competitive Programming Learning Path
+## Contributers
+
+Vedansh Priyadarshi [vedansh.priyadarshi@gmail.com](vedansh.priyadarshi@gmail.com) 
+
+Gagandeep Singh [czgdp1807@gmail.com](czgdp1807@gmail.com)
+
```
Comment (czgdp1807) [inline _posts/CPLP.md]: Add your full name with email.

File: _posts/CPLP.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+***
+#   Competitive Programming Learning Path
+## Contributers
+
+Vedansh Priyadarshi [vedansh.priyadarshi@gmail.com](vedansh.priyadarshi@gmail.com) 
+
+Gagandeep Singh [czgdp1807@gmail.com](czgdp1807@gmail.com)
```
Comment (czgdp1807) [inline _posts/CPLP.md]: `czgdp1807@gmail.com` -> `singh.23@iitj.ac.in`

File: _posts/CPLP.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
+
+
+***
+#   Competitive Programming Learning Path
+## Contributers
```
Comment (czgdp1807) [inline _posts/CPLP.md]: Contributers -> Contributors

File: _posts/CPLP.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
+
+
+***
```
Comment (czgdp1807) [inline _posts/CPLP.md]: Delete empty lines 1, 2, and 3.

**Comment by czgdp1807** [comment]
Thanks for the PR. 
A general comment - Rename the file to `2019-06-15-CPLP.md`.
Add the YAML front matter too.  See other blog posts.
In the author's section of the YAML front matter, add Vedansh Priyadarshi. Though these changes aren't compulsory as I will make some more edits shortly.

File: _posts/CPLP.md (hunk 1 above)
Comment (harshaldesai01) [reply _posts/CPLP.md]: Harshal Desai [harshaldesai01@gmail.com](harshaldesai01@gmail.com)

File: _posts/CPLP.md (hunk 2 above)
Comment (harshaldesai01) [reply _posts/CPLP.md]: Gagandeep Singh [singh.23@iitj.ac.in](singh.23@iitj.ac.in)

**Comment by czgdp1807** [comment]
Thanks for your contribution.

**Pull request merged**